py/tests/unit/with_runtime_sparkling/test_gbm.py

```diff
@@ -44,3 +50,81 @@ def testLoadAndTrainMojo(prostateDataset):
     assert len(predMojo) == len(predModel)
     for i in range(0, len(predMojo)):
         assert predMojo[i] == predModel[i]
+
+
+@pytest.fixture(scope="module")
+def dataset(spark):
+    return spark \
+        .read.csv("file://" + unit_test_utils.locate("smalldata/insurance.csv"), header=True, inferSchema=True) \
+        .withColumn("Offset", log(col("Holders")))
```

Comment:
Just question: In H2O examples I see that they usually create offset column from the response column, what was the reason to create it from `Holders` col? Just interested :)

Reply:
I've got inspired in this test https://github.com/h2oai/h2o-3/blob/master/h2o-py/tests/testdir_algos/gbm/pyunit_offset_tweedie_gbm.py#L14 testing correctness of a binary model.

Reply:
I see, thanks!